### ISSUE TITLE
FAQ Update

### ISSUE THREAD

**Issue by FozAhm** [FozAhm/quill#38]
I have to define the text but we have to add a FAQ for French people letting them know there is no problem if they speak French and that most us are fully bilingual. This is important cause we will be adverting to French CEGEPS.

**Comment by qiqi0725** [comment]
@FozAhm adding a french FAQ but the rest of the website is still in english?

**Comment by FozAhm** [comment]
Yes no need to translate everything, we just need one FAQ in French to let French speaking people its cool if they sing up and come. But the website in its entirety is in English otherwise.
